From ccfc866db68eeaaeae8665baf5345debe6f26e94 Mon Sep 17 00:00:00 2001
From: Jonas Gorski <jogo@openwrt.org>
Date: Sat, 18 May 2013 11:25:22 +0200
Subject: [PATCH 2/2] compat: handle renaming of random32() to prandom_u32()

496f2f93b1cc286f5a4f4f9acdc1e5314978683f renamed random32() to
prandom_u32(), so add an appropriate macro for older kernels.

Signed-off-by: Jonas Gorski <jogo@openwrt.org>
---
 include/linux/compat-3.10.h |   12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/include/linux/compat-3.10.h b/include/linux/compat-3.10.h
index 9d6e55c..1baa996 100644
--- a/include/linux/compat-3.10.h
+++ b/include/linux/compat-3.10.h
@@ -75,6 +75,18 @@ static inline void fb_enable_skip_vt_switch(struct fb_info *info)
 	module_driver(__pcmcia_driver, pcmcia_register_driver, \
 			pcmcia_unregister_driver)
 
+/**
+ * backport of:
+ *
+ * commit 496f2f93b1cc286f5a4f4f9acdc1e5314978683f
+ * Author: Akinobu Mita <akinobu.mita@gmail.com>
+ * Date:   Mon Dec 17 16:04:23 2012 -0800
+ *
+ *    random32: rename random32 to prandom
+ *
+ */
+#define prandom_u32()	random32()
+
 #else /* kernel is >= 3.10 */
 /*
  * We'd delete this upstream ever got this, we use our
-- 
1.7.10.4

